docs/user-guide/configure.md

```diff
@@ -109,6 +109,31 @@ For example, the following rule configuration would substitute in custom message
 
 Alternately, you can write a [custom formatter](../developer-guide/formatters.md) for maximum control if you need serious customization.
 
+Experimental feature: some rules support message arguments. For example, when configuring the `color-no-hex` rule, the hex color can be used in the message string:
+
+`.stylelintrc.js`:
+
+```js
+{
+  'color-no-hex': [true, {
+    message: (hex) => `Don't use hex colors like "${hex}"`,
+  }]
+}
+```
+
+`.stylelintrc.json`:
+
+<!-- prettier-ignore -->
```

Comment:
[note] Unless the disabling comment, Prettier would format the snippet like this:

```json
{
  "color-no-hex": [
    true,
    {
      "message": "Don't use hex colors like \"%s\""
    }
  ]
}
```

Compared to the snippet above for `.stylelintrc.js`, it would not be easier to read.